blockchain/state_manager/src/lib.rs

```diff
@@ -1291,7 +1291,7 @@ where
         Ok(out)
     }
 
-    /// Similar to `resolve_to_key_addr` in the vm crate but does not allow `Actor` type of addresses.
+    /// Similar to `resolve_to_key_addr` in the VM crate but does not allow `Actor` type of addresses.
```

Comment:
This should be `forest_vm` crate. It's not short for virtual machine, it's an actual name.